### ISSUE TITLE
Can not unmarshal bigint into *float64

### ISSUE THREAD

**Issue by HadesArchitect** [HadesArchitect/GrafanaCassandraDatasource#80]
**Setup**
```
create keyspace test with replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
create table test.employee ( emp_id text, emp_name text, login_time timestamp, PRIMARY KEY (emp_id));
insert into test.employee (emp_id, emp_name, login_time) values ('id1', 'Empl1', '2021-02-14T18:54:13+00:00');
insert into test.employee (emp_id, emp_name, login_time) values ('id2', 'Empl2', '2021-02-14T17:54:13+00:00');
insert into test.employee (emp_id, emp_name, login_time) values ('id3', 'Empl3', '2021-02-14T16:54:13+00:00');
```

**Query**
```
SELECT emp_id, COUNT(*), login_time FROM test.employee WHERE emp_id IN ('emp1', 'emp2', 'emp3') AND login_time > $__timeFrom and login_time < $__timeTo allow filtering
```

**Logs**
```
[DEBUG] cassandra-backend-datasource: Executing CQL query: 'SELECT emp_id, COUNT(*), login_time FROM test.employee WHERE emp_id IN ('emp1', 'emp2', 'emp3') AND login_time > 1613307990119 and login_time < 1613329590119 allow filtering' ..." 
[ERROR] cassandra-backend-datasource: Error while processing a query: can not unmarshal bigint into *float64" 
```

**Comment by HadesArchitect** [comment]
Similar issue `can not unmarshal varchar into *float64` with the query `SELECT emp_id, emp_id, login_time FROM test.employee WHERE emp_id IN ('id1', 'id2', 'id3') AND login_time > $__timeFrom and login_time < $__timeTo allow filtering
`

**Comment by HadesArchitect** [comment]
The second SELECT expression must be a valid double. Atm we expect user to use 'CAST(x AS DOUBLE)' always. Could there be a better way to handle it or should we just emphasize it in the documentation? @futuarmo ?

**Comment by HadesArchitect** [comment]
Not a bug, just needs proper error handling

**Comment by futuarmo** [comment]
Yes, I think we just should emphasize it in the documentation. Also we can use placeholder for notice about that
Any other way to fix it will be too complex or too specific for that task

**Comment by unflag** [comment]
Right!

**Comment by HadesArchitect** [comment]
Yeah!

**Comment by HadesArchitect** [comment]
Closed by #172, to be released as 3.0.0
